sdk/pinocchio/src/sysvars/slot_hashes/raw.rs

```diff
@@ -0,0 +1,116 @@
+//! Raw / caller-supplied buffer helpers for the SlotHashes sysvar.
```

Comment:
Maybe with the relaxed validation, this probably can be moved to the `impl<'a> SlotHashes<Ref<'a, [u8]>>`. It will be easier to "find" these helpers there.

Reply:
So, this file is for a different audience; the main functions in mod.rs are for when you already have the sysvar `AccountInfo` and want to create a `SlotHashes` wrapper around its data, but the functions in raw.rs are for fetching the sysvar data into a caller-provided buffer.

I moved this out at Illia's suggestion here: https://github.com/anza-xyz/pinocchio/pull/152#discussion_r2159787531

Reply:
Ah, got it. Makes sense. Can we move the `mod` and `pub use` definition of the module `raw` together with the others at the beginning of the file?